src/poliastro/plotting/porkchop.py

```diff
@@ -138,6 +138,10 @@ def porkchop(
 
     Returns
     -------
+    dv_launch: np.ndarray
+    Launch delta V 
+    dv_arrival: np.ndarray
+    Arrival delta v
```

Comment:
See line 145 and 146 and try to indentate like that.

Reply:
Thanks for the suggestion, but i figured out myself. Actually there was some other problem not this. My editor was making some problems and i had to change some settings on it and then it worked out :)